cluster-autoscaler/core/scale_up.go

```diff
@@ -63,14 +64,14 @@ func ScaleUp(context *context.AutoscalingContext, processors *ca_processors.Auto
 	nodeInfos, err := GetNodeInfosForGroups(nodes, context.CloudProvider, context.ClientSet,
 		daemonSets, context.PredicateChecker)
 	if err != nil {
-		return false, err.AddPrefix("failed to build node infos for node groups: ")
+		return false, &status.ScaleUpStatus{}, err.AddPrefix("failed to build node infos for node groups: ")
```

Comment:
nit: I think we could just return a nil status on error

Reply:
Done